data_format.html

```diff
@@ -229,7 +229,7 @@ <h3>
 		<div class="clear"></div>
 	</div>
 	<div id="footer">
-		<p>Copyright &copy; 2007 <a href="http://www.kitware.com">Kitware Inc</a>.</p>
+		<p>Copyright &copy; 2022 <a href="http://www.kitware.com">Kitware Inc</a>.</p>

```

Comment:
Why not (c) 2007-2022?

Reply:
No real reason. We can change it to that.

Reply:
I suggest to simplify maintenance and remove the year

>  Year of publication? This is unlikely to be worth the trouble to manually maintain in source files; source management tools provide more accurate information at lower resource cost.

Source: https://opensource.com/article/20/10/copyright-notices-open-source-software

Reply:
I agree let's remove the year.

Reply:
Yes, removing the year is fine, especially in the light of the article linked by JC.